src/sentry/runner/initializer.py

```diff
@@ -402,13 +402,19 @@ def validate_options(settings):
 def __model_unpickle_compat(model_id, attrs=None, factory=None):
     from django import VERSION
 
+    if attrs is not None or factory is not None:
+        metrics.incr("django.pickle.loaded_19_pickle.__model_unpickle_compat", skip_internal=False)
+
     if VERSION[:2] in [(1, 10), (1, 11)]:
         return model_unpickle(model_id)
     else:
         raise NotImplementedError
 
 
 def __simple_class_factory_compat(model, attrs):
```

Comment:
Only older models look for this factory, so if we go in here then we're loading an older model. Not totally sure this is right, maybe all that matters is that this function exists, and it will never get called in modern django?

Reply:
https://github.com/getsentry/sentry/pull/17003#discussion_r379143338